UberSignals.podspec

```diff
@@ -9,10 +9,13 @@ Pod::Spec.new do |s|
   s.source = { :git => 'https://github.com/uber/signals-ios.git', :tag => s.version }
   s.requires_arc = true
 
-  s.ios.deployment_target = '7.0'
+  s.ios.deployment_target = '8.0'
```

Comment:
Fair enough. Integrating this as a framework required 8.0, but integrating via a pod lib works with 7.0. Lets keep this as 8.0